### PR TITLE
README updated, unix install shifts to another file

### PR DESCRIPTION
**Pull request by david-puglielli**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/683)

### CODE REVIEW

**Comment by david-puglielli** [comment]
Debian link under Getting Started does not exist. There should be one if we support Debian and are pointing users to those pages

File: README.md [hunk 1]
```diff
@@ -43,433 +44,67 @@ Thank you for taking the time to participate in our last survey. You can continu
...
-For details, please read the documentation and/or take a look at the sample [build scripts](https://github.com/Microsoft/msphpsql/tree/dev/buildscripts#windows).
-
-## Install (Windows)
-
-#### Prerequisites
+For full details on the requirements for the drivers, see the [system requirements](https://docs.microsoft.com/en-us/sql/connect/php/system-requirements-for-the-php-sql-driver) on MSDN.
 
+On the client machine:
+- PHP 7.0.x, 7.1.x, or 7.2.1 and up on Windows, or 7.2.0 and up on Unix systems 
 - A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
```
Comment (yitam) [inline README.md]: please rephrase your sentence, or else it reads like we don't support 7.0* or 7.1* in Unix

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# Microsoft Drivers for PHP for SQL Server
+# Microsoft Drivers for PHP for Microsoft SQL Server
 
-**Welcome to the Microsoft Drivers for PHP for SQL Server PHP 7**
+**Welcome to the Microsoft Drivers for PHP for Microsoft SQL Server**
 
-The Microsoft Drivers for PHP for SQL Server are PHP extensions that allow for the reading and writing of SQL Server data from within PHP scripts. The SQLSRV extension provides a procedural interface while the PDO_SQLSRV extension implements PDO for accessing data in all editions of SQL Server 2008 R2 and later (including Azure SQL DB). These drivers rely on the Microsoft ODBC Driver for SQL Server to handle the low-level communication with SQL Server.
+The Microsoft Drivers for PHP for Microsoft SQL Server are PHP extensions that allow for the reading and writing of SQL Server data from within PHP scripts. The SQLSRV extension provides a procedural interface while the PDO_SQLSRV extension implements PHP Data Objects (PDO) for accessing data in all editions of SQL Server 2008 and later (including Azure SQL DB). These drivers rely on the Microsoft ODBC Driver for SQL Server to handle the low-level communication with SQL Server.
```
Comment (yitam) [inline README.md]: SQL Server 2008 R2

File: README.md [hunk 3]
```diff
@@ -43,433 +44,67 @@ Thank you for taking the time to participate in our last survey. You can continu
...
-    
-    
-### Step 5: Restart Apache to load the new php.ini file
-
-**Ubuntu, Debian, and SUSE**
-
-    sudo systemctl restart apache2
+The drivers are distributed as pre-compiled extensions for PHP found on the [releases page](https://github.com/Microsoft/msphpsql/releases). They are available in thread-safe and non thread-safe versions, and in 32-bit and 64-bit versions. The source code for the drivers is also available, and you can compile them as thread safe or non-thread safe versions. The thread safety configuration of your web server will determine which version you need. 
+ 
+If you choose to build the drivers, you must be able to build PHP 7 without including these extensions. For help building PHP on Windows, see the [official PHP website][phpbuild]. For details on compiling the drivers, see the [documentation](https://github.com/Microsoft/msphpsql/tree/dev/buildscripts#windows) -- a buildscript is provided, but you can also compile the drivers manually.
```
Comment (yitam) [inline README.md]: You may want to say an example build script

File: README.md [hunk 4]
```diff
@@ -43,433 +44,67 @@ Thank you for taking the time to participate in our last survey. You can continu
...
  
-version number MAY have trailing pre-release version to indicate the stability, and/or build meta data.
-
-- Pre-release version is denoted by hyphen followed by `preview` or `rc` keyword and may be followed by a series of dot separated identifiers. Production quality releases do not contain the pre-release version. `preview` has lower precedence than `rc`. Example of precedence: *preview < preview.1 < rc < rc.1*. 
-*Note that PECL package version does not have the hyphen before pre-release version, due to restrictions in PECL. Example of PECL package version: 1.2.3preview*
-- Build metadata MAY be denoted by a plus sign followed by 4 digits, such as  `1.2.3-preview+5678` or `1.2.3+5678`. Build meta data does NOT figure into the precedence order.
+The version number may have trailing pre-release version identifiers to indicate the stability and/or build metadata.
 
-    
+- Pre-release version is denoted by a hyphen followed by `preview` or `rc` and may be followed by a series of dot separated identifiers. Production quality releases do not contain the pre-release version. `preview` has lower precedence than `rc. Example of precedence: *preview < preview.1 < rc < rc.1*.
```
[the remark this answers is not in the document]
Comment (yitam) [reply README.md]: I think you forgot to close `rc

File: README.md (hunk 1 above)
Comment (lilgreenbird) [inline README.md]: we do support 7.0 and 7.1 on unix

File: README.md [hunk 5]
```diff
@@ -43,433 +44,67 @@ Thank you for taking the time to participate in our last survey. You can continu
...
 - A Web server such as Internet Information Services (IIS) is required. Your Web server must be configured to run PHP
-- [Microsoft ODBC Driver 11][odbc11], [Microsoft ODBC Driver 13][odbc13] or [Microsoft ODBC Driver 17][odbc17]
-
-#### Enable the drivers
-
-1. Make sure that the driver is in your PHP extension directory (you can simply copy it there if you did not use nmake install).
+- [Microsoft ODBC Driver 11][odbc11], [Microsoft ODBC Driver 13][odbc13], or [Microsoft ODBC Driver 17][odbc17]
 
-2. Enable it within your PHP installation's php.ini: `extension=php_sqlsrv.dll` and/or `extension=php_pdo_sqlsrv.dll`.  If necessary, specify the extension directory using extension_dir, for example: `extension_dir = "C:\PHP\ext"`. Note that the precompiled binaries have different names -- substitute accordingly in php.ini.
+On the server side, SQL Server 2008 and above on Windows is supported, as is SQL Server 2016 and above on Linux.
```
Comment (lilgreenbird) [inline README.md]: MS SQL Server

Comment (lilgreenbird) [reply README.md]: and 2008 R2

File: README.md [hunk 6]
```diff
@@ -478,11 +112,11 @@ Thank you!
 ## FAQs
 **Q:** Can we get dates for any of the Future Plans listed above?
 
-**A:** At this time, Microsoft is not able to announce dates. We are working extremely hard to release future versions of the driver. We will share future plans as appropriate. 
+**A:** At this time, Microsoft is not able to announce dates. We are working hard to release future versions of the driver and will share future plans as appropriate. 
 
 **Q:** What's next?
 
-**A:** On July 6, 2017 we released the production release version 4.3.0 of our PHP Driver. We will continue working on our future plans and releasing previews of upcoming releases frequently.
+**A:** On February 6, 2018 we released the production release version 5.2.0 of our PHP Driver. We will continue working on our future plans and releasing previews of upcoming releases frequently.
```
Comment (yitam) [inline README.md]: The release date is still up in the air... :(

Comment (david-puglielli) [reply README.md]: I'll update once we know the exact date.

**Comment by david-puglielli** [comment]
Snapshot file has broken links to the ODBC install page on MSDN.

File: Linux-mac-install.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
+# PHP Linux and Mac Drivers Installation Tutorial
+The following instructions assume a clean environment and show how to install PHP 7.x, the Microsoft ODBC driver, Apache, and the Microsoft drivers for PHP for Microsoft SQL Server on Ubuntu 16.04 and 17.10, RedHat 7, Debian 8 and 9, Suse 12, and macOS X 10.11 and 10.12. These instructions advise installing the drivers using PECL, but you can also download the prebuilt binaries from the [Microsoft Drivers for PHP for Microsoft SQL Server](https://github.com/Microsoft/msphpsql/releases) Github project page and install them following the instructions in [Loading the Microsoft Drivers for PHP for Microsoft SQL Server](../../connect/php/loading-the-php-sql-driver.md). For an explanation of extension loading and why we do not add the extensions to php.ini, see the section on [loading the drivers](../../connect/php/loading-the-php-sql-driver.md##loading-the-driver-at-php-startup).
```
Comment (lilgreenbird) [inline Linux-mac-install.md]: it's macOS 10.11 and 10.12 (no X)

Comment (david-puglielli) [reply Linux-mac-install.md]: of/on are both commonly used, but dictionaries seem to give examples using of.

**Pull request merged**